### DISCUSSION TITLE
improvement for reused passwords feature

### DISCUSSION THREAD

**Discussion by emhl** [dani-garcia/vaultwarden#2677] [Ideas]
In the `Report` > `Reused Passwords` tab it shows how many times the password of this account has been used. but i don't know which accounts share the identical password without comparing them manually. 

it would be nice to either be able to group the accounts by "same password" or show which accounts share the same password when clicking on one

**Comment by BlackDex** [comment]
While this sound like a nice feature, that is not something done server side, which is what Vaultwarden is.
The client side is fully Bitwarden (with some small modifications so that it works with Vaultwarden).

It also looks like there is a topic on this at Bitwarden already: https://community.bitwarden.com/t/group-exposed-password-reused-password-and-weak-password-lists-by-password/15485
